source/package.rst

```diff
@@ -0,0 +1,689 @@
+
+*   Opret et moderne Python-projekt
+
+    *   Brug ``pyproject.toml``
+
+        *   Denne konfigurationsfil bruges til basal opsætning af projektet og
+            de værktøjer, der kan læse deres konfigurationer i denne,
+            eksempelvis ``black`` og ``pytest``.
+
+    *   Brug ``setup.cfg``
+
+        *   Bemærk, at der efter ``setuptools>=43.0.0`` ikke er behov for en
+            ``setup.py``-fil.
+
+
+
+
+Mappestruktur og konfigurationer
+================================
+
+Generelt er valget af mappestruktur vigtig for understøttelse de forskellige
+processer i projekt-livscyklussen. Kildekoden er versionsstyret sammen med alt
+andet kildemateriale i en versionsstyret projekt-mappe på GitHub. Hver
+koderevision har tilknyttet dokumentation og testfunktionalitet, samt al anden
+konfiguration til proces-understøttelse. Formålet med at have alt samlet er, at
+hvor alle komponenter i projektet følges ad og for en given Git-revision
+fungerer sammen.
+
+Konsekvensen af denne fremgangsmåde er blandt andet, at der for en given
+revision af koden ikke bare findes tests, der dækker koden, men også tilhørende
+dokumentation, der beskriver funktionaliteten i den pågældende revision.
+
+Følgende er et eksempel på mappestrukturen for en færdig Python-pakke i et
+Git-arkiv:
+
+.. code-block:: none
+
+    package
+    ├── .git
+    ├── .github
+    │   └── workflows
+    │       └── main.yaml
+    ├── .gitignore
+    ├── docs
+    │   ├── index.md
+    │   └── ...
+    ├── environment-dev.yml
+    ├── environment.yml
+    ├── LICENSE
+    ├── mkdocs.yml
+    ├── pyproject.toml
+    ├── README.md
+    ├── scripts
+    │   ├── examples
+    │   │   └── example1.py
+    │   └── ci
+    │       └── some_behaviour.sh
+    ├── setup.cfg
+    ├── src
+    │   └── package
+    │       ├── module1.py
+    │       ├── module2.py
+    │       └── __init__.py
+    └── tests
+        └── package
+            ├── test_module1.py
+            └── test_module2.py
+
+Bemærkninger:
+
+*   Mappen med Python-pakkens kildekode ligger adskilt fra roden i en separat
+    mappe `src`.
+
+*   Test-funktionalitet er adskilt fra kildekoden, som den tester, så den ikke
+    installeres sammen med pakken.
+
+    *   Brugeren bør ikke have nogen grund til at teste koden.
+    *   Pakke bør ikke have en masse overflødig funktionalitet med, herunder
+        test-data.
+
+*   Dokumentations-materiale ud over README-filen og eventuelle små-filer til
+    dokumentation, ligger i sin egen mappe af samme årsag som med kildekoden:
+    Dokumentationen skal også bygges og udgives.
+
+*   Alle andre mapper og filer er konfigurationer, scripts og andet til brug
+    primært for udviklere samt for brugeren, der skal installere pakken ud fra
+    arkivet.
+
+
+Udviklerens synsvinkel
+----------------------
+
+| Som udvikler
+| skal jeg have nogle byggematerialer,
+| der gør det muligt at bygge, dokumentere og kvalitetssikre pakken.
```

Comment:
Efter at have læst forfra en del gange forstår jeg nogenlunde hvad du vil med den her kontstruktion. Jeg synes det var svært at læse første gang, jeg snublede en del over det og blev forvirret. Jeg tror det kan afhjælpes med anderledes formatering, fx
```suggestion
    | *Som udvikler*
    | *skal jeg have nogle byggematerialer,*
    | *der gør det muligt at bygge, dokumentere og kvalitetssikre pakken.*
```
Det gælder også for den tilsvarende tekst i næste afsnit